### PR TITLE
[M10-400] Ogury SDK Integration

### PR DESCRIPTION
**Pull request by carlosalcala-freestar**
**[M10-364](https://freestar.atlassian.net/browse/M10-364)**
**[M10-400](https://freestar.atlassian.net/browse/M10-400)**
**[M10-401](https://freestar.atlassian.net/browse/M10-401)**

## Description, Motivation and Context

**What is the goal of ticket?  What are the acceptance criteria?**

Integrate Ogury SDK - Banners
Integrate Ogury SDK - Interstitials
Integrate Ogury SDK - Rewards

## How Has This Been Tested? 

Manually tested App Build 

## Does this change require updates outside of the code, like updating the documentation, adding a feature flag, or adding fields to a DB table? If so, was this done?

N/A

## Any tickets that this is dependent upon?

N/A

## Should reviewers manually test your changes?

No

### Evidence

Screen Shot 2022-08-05 at 18 16 56

### CODE REVIEW

File: FreestarAds-Ogury/Source/objc/OguryFSTRPartnerInitializer.m [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        instance = [[OguryFSTRPartnerInitializer alloc] init];
+    });
+    return instance;
+}
+
+- (void)runInitialization {
+    // run partner initialization here
+    FSTRLog(@"OGURY: Partner initialization.");
+
+    OguryConfigurationBuilder *configurationBuilder = [[OguryConfigurationBuilder alloc] initWithAssetKey:@"OGY-374D2BA758F4"];
```
Comment (maddapper) [inline FreestarAds-Ogury/Source/objc/OguryFSTRPartnerInitializer.m]: @carlosalcala-freestar are you sure you want hardcode the asset key?  do we hardcode asset keys in Android?

**Review by maddapper — COMMENTED** [review]
@carlosalcala-freestar not sure if hardcoding keys is a good approach, does Android hardcode asset keys for ogury?

File: FreestarAds-Ogury/Source/objc/OguryFSTRPartnerInitializer.m (hunk 1 above)
Comment (carlosalcala-freestar) [reply FreestarAds-Ogury/Source/objc/OguryFSTRPartnerInitializer.m]: @maddapper do we have the Ogury Asset Key in one of the Mediator/Adapter class properties ? In Yahoo we use the app_id but not sure in this case 🤔

**Review by maddapper — APPROVED** [review]
@carlosalcala-freestar i would make sure to test this again, to make sure the changes didn't break anything.

**Review by maddapper — APPROVED** [review]

**Pull request merged**